src/main/java/org/reactome/addlinks/linkchecking/LinkChecker.java

```diff
@@ -69,9 +69,9 @@ public LinkCheckInfo checkLink() throws HttpHostConnectException, IOException, E
 											.setRelativeRedirectsAllowed(true)
 											.setConnectionRequestTimeout(1000 * (int)this.timeout.getSeconds()).build();
```

Comment:
The indents here (and from line 65) are off

Reply:
Yes, github's default tabsize is too large. Appending "?ts=4" to the URL can fix it.